### PR TITLE
Adding Version url with version number and git sha

### PR DESCRIPTION
**Pull request by fabricematrat**
QA:
make server, go to http://localhost:8080/v4/debug/info
Check that version and id are unknown and that you get a log saying that the version file is not found

Add a version.yaml in cm/charmd/ with version: something and id: something 
Check again it is displayed and no log are appearing.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/608/

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/609/

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/610/

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/611/

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/612/

File: version/version.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package version
```
Comment (urosj) [inline version/version.go:4]: Should we add Copyright notice?

**Comment by urosj** [comment]
:+1:

File: version/version.go (hunk 1 above)
Comment (fabricematrat) [inline version/version.go:4]: Added Copyright

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/626/

**Comment by jrwren** [comment]
LGTM, QAed on the deb PR

**Comment by fabricematrat** [comment]
:shipit: 
Thanks @jrwren && @urosj && @rogpeppe

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**